pkgs/development/libraries/qt-5/modules/qtwebkit.nix

```diff
@@ -54,9 +54,9 @@ qtModule {
     [
       # with gcc7 this warning blows the log over Hydra's limit
       "-Wno-expansion-to-defined"
```

Comment:
But this one gets ignored?

Reply:
No, this one is actually supported in clang: https://clang.llvm.org/docs/DiagnosticsReference.html#wdisabled-macro-expansion

Reply:
To clarify my phrasing/use of "ignore" - with GCC it's the default to ignore non-existent warning options. With clang it's afaik not. There is `-Wno-unknown-warning-option` for that, which we don't use as far as I can tell.